### PR TITLE
[Article] React hooks color picker [DO NOT MERGE UNTIL 12/5]

### PR DESCRIPTION
**Pull request by shin10kudev**

### CODE REVIEW

File: src/content/react-hooks-color-picker.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+```js
+<ColorBox currentColor={currentColor} />
+```
+
+## Summing up
+
+And there you have it! With a few lines of code you have your own custom color picker made with react.
+
+Of course, `useState` is not the only React Hook out there. To go further with your React Hooks studies, please check out the official <a href="https://reactjs.org/docs/hooks-intro.html" target="_blank" rel="noopener noreferrer">React Hooks docs</a>at reactjs.org.
```
Comment (yosi-q) [inline src/content/react-hooks-color-picker.md:164]: @eliotc1986 

Why you use not Markdown writing rule but html a tag?
If you want to add attribute "target" and "rel", I think it's better to use "gatsby-remark-external-links".
Take a look at following site: https://pastora.jp/posts/set-target-blank-to-the-external-link/

**Review by yosi-q — COMMENTED** [review]
I wrote a comment regarding anchor link.
Please have a look and think on it.

Thank you

File: src/content/react-hooks-color-picker.md (hunk 1 above)
Comment (shin10kudev) [reply src/content/react-hooks-color-picker.md:164]: @yosi-q Thank you for the comment!
For this, I don't want to add another dependency. Plus, Markdown is fully capable of handling html5 syntax. https://www.markdownguide.org/basic-syntax/

So, is it okay if I skip this?

**Review by yosi-q — APPROVED** [review]
LGTM latest version!

**Pull request merged**